.travis.yml

```diff
@@ -5,7 +5,7 @@ node_js:
   - "10"
 deploy:
   provider: script
-  script: yarn install && yarn run deploy
+  script: yarn install && yarn format:check && yarn run deploy
```

Comment:
we want `format:write`, don't we?

Reply:
actually I'm not 100% sure what you're trying to do here. I'm busy right now, but maybe I can have a go at lunchtime. what we want, I think, is a pre-commit hook.

Reply:
https://prettier.io/docs/en/precommit.html

Reply:
I think it should be the contributor's responsibility to commit changes according to the repositories style guide. How they do it should be up to the individual.

Having that said, the purpose of this PR is just to enforce these rules through an additional check. Therefore I've added the `yarn format:check` cmd which will return a non-zero exit code if the commit is not aligned with the style guide.